sentry-spring/src/main/java/io/sentry/spring/SentryRequestFilter.java

```diff
@@ -29,11 +31,23 @@ protected void doFilterInternal(
       final @NotNull FilterChain filterChain)
       throws ServletException, IOException {
     hub.pushScope();
+    // clears breadcrumbs that may have been added during application startup through one of the logging integrations.
+    hub.clearBreadcrumbs();
```

Comment:
would it make sense to expose the `clear` method in the hub then? which is basically cleaning everything from the `scope`?

Reply:
could we lose useful information when cleaning all the breadcrumbs?

Reply:
This is there to avoid having startup logs attached to every event sent. If we do not clear breadcrumbs every event looks like: 
![image](https://user-images.githubusercontent.com/1357927/92219014-3d5b7680-ee9a-11ea-8e58-f95e8f87bcaa.png)

Reply:
IMO this is fine, not sure how noisy this can get though.
every event on android also has many startup breadcrumbs, but I believe that this is what breadcrumbs stand for, if it's like less than 5, all good, it'd be a problem if its a huge number and just trash.
maybe @bruno-garcia  has opinions
I'm just afraid of losing useful breadcrumbs added by the user manually.

Reply:
I still believe these breadcrumbs create noise, but since both @marandaneto and @bruno-garcia are more leaning towards keeping them, they are now not removed.